services/src/test/scala/cromwell/pipeline/service/GitLabProjectVersioningTest.scala

```diff
@@ -96,6 +96,54 @@ class GitLabProjectVersioningTest
       }
     }
 
+    "getUpdatedProjectVersion" should {
+      val userVersion = PipelineVersion("v0.0.1")
+      val project = TestProjectUtils.getDummyProject()
+      def request(project: Project): Future[Response[Seq[GitLabVersion]]] =
+        mockHttpClient.get[Seq[GitLabVersion]](
+          url = exact(gitLabConfig.url + "projects/" + project.repositoryId.value + "/repository/tags"),
+          headers = any[Map[String, String]],
+          params = any[Map[String, String]]
+        )(any[ExecutionContext], any[Reads[Seq[GitLabVersion]]])
+
+      "return correct PipelineVersion" taggedAs Service in {
```

Comment:
Try to avoid vague phrases like `return correct X`. What do you mean by `correct PipelineVersion `? Is it the latest one? Is it the version that user provided?